src/lib/tabs/tab-body.ts

```diff
@@ -53,6 +59,44 @@ export type MdTabBodyPositionState =
 export type MdTabBodyOriginState = 'left' | 'right';
 
 /**
+ * The portal host directive for the contents of the tab.
+ * @docs-private
+ */
+@Directive({
+  selector: '[matTabBodyHost]'
+})
+export class MdTabBodyPortal extends _MdTabBodyPortalBaseClass implements OnInit, OnDestroy {
+  /** A subscription to events for when the tab body begins centering. */
+  private _centeringSub: Subscription;
+
+  constructor(
+    _componentFactoryResolver: ComponentFactoryResolver,
+    _viewContainerRef: ViewContainerRef,
+    private _host: MdTabBody) {
```

Comment:
@josephperrott it looks like we need to use a forwardRef here since this directive is used in the template for `MdTabBody`. Here's an example for a similar situation in the sidenav: https://github.com/angular/material2/blob/master/src/lib/sidenav/sidenav.ts#L36

Reply:
Done